wpeverest/user-registration
# Enhance / Keyboard Shortcuts Dialog

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds the Keyboard Shortcuts Help Dialog to inform users about the available Hotkeys we have provided.

Closes # .

### How to test the changes in this Pull Request:

1. Keyboards Shortut dialog box should appear everytime a new form is created.
2. Pressing `Ctrl + H` should display the Keyboard Shortcut dialog box
3. There is a quick link at bottom right in Form Edit page. Clicking on *Keyboard Shortcut* link should also open Keyboard Shortcuts dialog box.

### Types of changes:




### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
